Docs/Contributing.md

```diff
@@ -28,15 +28,15 @@ If you'd like to improve the [documentation](./), [manual](../Source/Documentati
 
 There are no requirements for the pull request.
 
-### Bug process
+### Process for bugs and other issues
 
-If you'd like to fix a bug, you can get started immediately. If the fix turns out to be very small, you do not even need a bug report. Otherwise, you will need to make sure it has been reported on [our bug tracker on Launchpad](https://bugs.launchpad.net/or). If it has not, you can report the bug *and* fix it!
+If you'd like to fix an issue, you can get started immediately. If the fix turns out to be very small, you do not even need an bug report. Otherwise, you will need to make sure it has been reported on [our issue tracker on GitHub](https://github.com/openrails/openrails/issues/). If it has not, you can report the issue *and* fix it!
```

Comment:
```suggestion
If you'd like to fix an issue, you can get started immediately. If the fix turns out to be very small, you do not even need a bug report. Otherwise, you will need to make sure it has been reported on [our issue tracker on GitHub](https://github.com/openrails/openrails/issues/). If it has not, you can report the issue *and* fix it!
```